l10n/de_DE/LC_MESSAGES/funding.po

```diff
@@ -91,6 +91,14 @@ msgstr "Ausbezahlter Betrag"
 msgid "The sum of the amounts of accepted drawdowns."
 msgstr "Die Summe der Beträge akzeptierter Mittelabrufe."
 
+#: Civi/Funding/Api4/Action/FundingCase/GetFieldsAction.php managed/SavedSearchFundingCases.mgd.php
+msgid "Review Progress"
+msgstr "Prüffortschritt"
+
+#: Civi/Funding/Api4/Action/FundingCase/GetFieldsAction.php
+msgid "The progress of application review in percent."
+msgstr "Zeigt den Fortschritt der Antragsprüfung in Prozent."
```

Comment:
I'd suggest to reduce this to `Der Fortschritt der Antragsprüfung in Prozent.` which would correspond to the English version.

Reply:
fixed